tests/testthat/test-utils.R

```diff
@@ -197,3 +197,15 @@ test_that("Error is thrown if colnames are missing", {
     "columns: D, E."
   )
 })
+
+# validate scenario_type-------------------------------------------------------
+test_that("Error is thrown if NGFS scenario is combined with non-NGFS scenario", {
```

Comment:
if you want to be really great also ad a unit test for when it words, i.e. that e.g. "is_ngfs" gets returned